openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
Okay, with the get_requirements I have a major problem.

option 1: getRequirements (all ) -- too much permission

option 2: removing speakerOverlay, and re-adding it as a `<div>` on the projector templates: complete rewrite, still needs to getRequirements(all usernames¹).

option3: option 2 + adding a field "user_name" to agenda_item.speakers and ...poll.options.

¹as every other projector slide displaying usernames, because the names will change before the projector is reloaded.